Input/Input.module.less

```diff
@@ -91,6 +96,19 @@
 			background-color: @agate-input-focused-background-color;
 			background-image: @agate-input-focused-background-image;
 			box-shadow: 0 0 0 @agate-input-decorator-focus-border-width @agate-input-decorator-focused-border-color;
+
+			.input {
+				caret-color: @agate-accent;
```

Comment:
This should refer to a new `@agate-input-caret-color` variable, defined in the colors files.

In `colors-base.less`:
```css
@agate-input-caret-color: @agate-accent;
```